runscanner-illumina/main.cpp

```diff
@@ -312,9 +312,8 @@ void add_lane_charts(
   add_table_column(columns, "Lane", "lane");
   add_table_column(columns, "Pool", "pool");
   add_table_column(columns, "Density %", "densityPct");
-  add_table_column(columns, "Density", "density");
+  add_table_column(columns, "Density (K clusters/mm²)", "density");
```

Comment:
I have great fear that C++'s Unicode support will break, but here's hoping.

Reply:
Wait, there's a UTF-8 literal support: https://en.cppreference.com/w/cpp/language/string_literal
Try making this `u8"Density (K clusters/mm²)"`